src/base/bittorrent/session.cpp

```diff
@@ -2015,7 +2283,11 @@ void Session::setEncryption(int state)
 {
     if (state != encryption()) {
         m_encryption = state;
-        configureEncryption();
+        configureDeferred();
+        Logger::instance()->addMessage(
+                    tr("Encryption support [%1]")
```

Comment:
What about the price? I gave you my motives. If you disagree, say it right, don't beat around the Bush.